libs/@guardian/core-web-vitals/src/index.ts

```diff
@@ -89,7 +89,7 @@ const getCoreWebVitals = async (): Promise<void> => {
 	const webVitals = await import('web-vitals');
 	const { getCLS, getFCP, getFID, getLCP, getTTFB } = webVitals;
```

Comment:
From the docs, we should update these methods to the new names:

> _**Note:** in version 2, these functions were named `getXXX()` rather than `onXXX()`. They've [been renamed](https://github.com/GoogleChrome/web-vitals/pull/222) in version 3 to reduce confusion (see [#217](https://github.com/GoogleChrome/web-vitals/pull/217) for details) and will continue to be available using the `getXXX()` until at least version 4. Users are encouraged to switch to the new names, though, for future compatibility._

Reply:
Ha I merged these and now CI fails